modules/stochastic_tools/doc/stm_workshop/intro_slides/parameter_study_workshop.md

```diff
@@ -0,0 +1,132 @@
+# Parameter Study Syntax
+
+!style halign=center
+[https://mooseframework.inl.gov/syntax/ParameterStudy/index.html](https://mooseframework.inl.gov/syntax/ParameterStudy/index.html)
+
+!---
+
+# ParameterStudy
+
+- Action that simplifies syntax for simple parameter studies
+- No need to identify value names or run into common, annoying input errors.
+- Programmatic creation of `Samplers`, `Distributions`, `Controls`, `Transfers`, `Reporters`, and `Outputs` objects.
+- Automatic multiapp mode detection
+
+!---
+
+!row!
+!col! width=48%
+```
+[StochasticTools]
+[]
+
+[Distributions]
+  [gamma]
+    type = Uniform
+    lower_bound = 0.5
+    upper_bound = 2.5
+  []
+  [q_0]
+    type = Weibull
```

Comment:
Could you link to existing test/example files? That way you won't have to change the slides every time a parameter changes.